### DISCUSSION TITLE
Checkboxes in list are unselected when I change filtering conditions

### DISCUSSION THREAD

**Discussion by Neboyarsky** [allure-framework/allure2#2415] [TestOps]
### Describe the Bug

Whenever I change filtering settings (i.e. update search phrase in "filter", or select Broken checkbox when I had only Failed selected, or something similar, the checkboxes in the list of tests are reset to unselected state.
If this is expected behaviour, I would be happy to hear a reasoning behind the decision to implement it that way.

### Steps to Reproduce

Open a report
Tick boxes on one or more test cases
Update filtering settings in the left panel in a way that some of selected tasks meet the filtering criteria (adding another filtering category will work, i.e. you are looking at Failed tests and tick Broken tests)

### Expected Behaviour

Checkboxes that are selected are reset to unselected state

### Screenshots or Additional Context

https://github.com/allure-framework/allure2/assets/44096384/1e0ec86c-1c9d-4e24-b60a-bafd10f7a1bf

### What Language are you using?

Java

### What Framework/Allure Integration you are using?

irrelevant

### What version of Allure Integration you are using?

irrelevant

### What version of Allure Report you are using?

irrelevant

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cheshi-mantu** [comment]
@Neboyarsky hi!
we accept bug reports for Allure Testops via https://help.qameta.io
Could you please register there using the email address provided by the company owning the licence for Allure Testops and file your bug report. We are going to need some additional information on the release you are currently using.

cheers

**Reply by Neboyarsky** [reply]
Hi @cheshi-mantu. If you folks do not accept bug reports here, then why do you have bugs section open for public? It would be way more user-friendly to just have a note here at gitbub that "hey folks, bugs are accepted there, not here". I feel villated by lack of such communication from the very beginning. Making to wait for a month and a half only to hear that "you should have submitted a bug elsewhere" is uncool.

**Reply by cheshi-mantu** [reply]
I feel your disappointment, sir.
When a company or a team buy Allure Testops licences, they are being informed that https://help.qameta.io is the way to get support. And this is only way for us to respond in timely manner.
This project is for the open source Allure Report which is developed and maintained by other team and supported by the community and Allure Report dev team. Allure Testops is a commercial product with dedicated dev, QA etc and support via dedicated channel different from Allure Report.
